src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+type TodosProps = {
+  todo: Todos
```

Comment:
there is only 1 task here, so name the types more properly
```suggestion
  todo: Todo,
```